lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -129,6 +129,7 @@ def buffers
       def self.create_later(attrs, storage_adapter)
         element = new(attrs)
         storage_adapter.buffers[:upsert][element.unique_identifier] = element
+        element
```

Comment:
it doesn't, just a stray commit when I was seeing some odd behavior in dalton. will reverse.